src/app/components/Resume/ResumePDF/ResumePDFProfile.tsx

```diff
@@ -52,18 +52,35 @@ export const ResumePDFProfile = ({
             }
           }
 
-          const shouldUseLinkWrapper = ["email", "url"].includes(key);
+          const shouldUseLinkWrapper = ["email", "url", "phone"].includes(key);
           const Wrapper = ({ children }: { children: React.ReactNode }) => {
             if (!shouldUseLinkWrapper) return <>{children}</>;
 
-            return (
-              <ResumePDFLink
-                src={key === "email" ? `mailto:${value}` : `https://${value}`}
-                isPDF={isPDF}
-              >
-                {children}
-              </ResumePDFLink>
-            );
+            switch (key) {
+              case "email":
+                return (
+                  <ResumePDFLink src={`mailto:${value}`} isPDF={isPDF}>
+                    {children}
+                  </ResumePDFLink>
+                );
+
+              case "phone":
+                return (
+                  <ResumePDFLink
+                    src={`tel:${value.replace(/[\(\)\s-]/g, "")}`} // This regex "cleans" the number
```

Comment:
nit: This should work great. An alternative is to make it more explicit and replace all non-numbers, e.g. `.replace(/[^0-9]/g, '')`

Reply:
I don't think this would work, since it would remove the + at the front which wouldn't allow designation of telephone codes. (I think that's what they're called?)

Reply:
Good catch on +